Buildings/Examples/FanCoils/FourPipe.mo

```diff
@@ -0,0 +1,696 @@
+    final nLay=3,
+    material={matWoo,matIns,matGyp})
+    "Exterior construction"
+    annotation (Placement(transformation(extent={{300,260},{320,280}})));
+
+  parameter HeatTransfer.Data.OpaqueConstructions.Generic conIntWal(
+    final nLay=1,
+    material={matGyp2})
+    "Interior wall construction"
+    annotation (Placement(transformation(extent={{338,260},{358,280}})));
+
+  parameter HeatTransfer.Data.OpaqueConstructions.Generic conFur(
+    final nLay=1,
+    material={matFur})
+    "Construction for internal mass of furniture"
+    annotation (Placement(transformation(extent={{338,220},{358,240}})));
+
+  parameter HeatTransfer.Data.OpaqueConstructions.Generic conFlo(
+    final nLay=1,
+    material={matCon})
+    "Floor construction (opa_a is carpet)"
+    annotation (Placement(transformation(extent={{300,220},{320,240}})));
+
+  Buildings.Fluid.Sources.Boundary_pT souHea(
+    redeclare package Medium = MediumW,
+    p(displayUnit="Pa") = 100000 + 3000,
+    T=333.15,
+    nPorts=1)
+    "Source for hot water"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}},
+      rotation=90, origin={70,-230})));
+
+  Buildings.Fluid.Sources.Boundary_pT sinHea(
+    redeclare package Medium = MediumW,
+    p=100000,
+    T=328.15,
+    nPorts=1)
+    "Sink for hot water"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}},
+      rotation=90, origin={20,-230})));
+
+  Buildings.Fluid.Sources.Boundary_pT sinCoo(
+    redeclare package Medium = MediumW,
+    p=100000,
+    T=288.15,
+    nPorts=3)
+    "Sink for chilled water"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}},
+      rotation=90, origin={110,-230})));
+
+  Buildings.Fluid.Sources.Boundary_pT souCoo(
+    redeclare package Medium = MediumW,
+    p(displayUnit="Pa") = 100000 + 3000,
+    T=279.15,
+    nPorts=3)
+    "Source for chilled water"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}},
+      rotation=90, origin={152,-230})));
+
+  Buildings.Fluid.ZoneEquipment.FourPipe fanCoiUni(
+    redeclare package MediumA = MediumA,
+    redeclare package MediumHW = MediumW,
+    redeclare package MediumCHW = MediumW,
+    heaCoiTyp=Buildings.Controls.OBC.ASHRAE.G36.Types.HeatingCoil.None,
+    QCoiCoo_flow_nominal=-20000,
+    dpAir_nominal=100,
+    mCoiCooWat_flow_nominal=4*0.2984,
+    dpCooCoiWat_nominal(displayUnit="Pa") = 1000,
+    mAir_flow_nominal=0.21303*2*3) "Fan coil unit with no heating coil"
+    annotation (Placement(transformation(extent={{28,140},{68,180}})));
+
+  Buildings.Controls.OBC.ASHRAE.G36.FanCoilUnits.Controller conFCU(
+    cooCoi=Buildings.Controls.OBC.ASHRAE.G36.Types.CoolingCoil.WaterBased,
+    heaCoi=Buildings.Controls.OBC.ASHRAE.G36.Types.HeatingCoil.None,
+    final TiCoo=200,
+    final heaConTyp=Buildings.Controls.OBC.CDL.Types.SimpleController.PI,
+    final kHea=0.05,
+    final TiHea=120,
+    final kCooCoi=0.05,
+    final TiCooCoi=90,
+    final kHeaCoi=0.005,
```

Comment:
This is unreasonably small. Why does it need to be that small? It essentially requires a control error of 1/0.005 = 200 Kelvin in order to fully activate the P controller. 
So this is essentially an I-controller, which is inherently sluggish and risks instability.

Reply:
Updated in #4371.